controllers/helpers.go

```diff
@@ -126,6 +129,31 @@ func validateLabelSelector(selector labels.Selector) error {
 	return nil
 }
 
+// transformCloudSpecToHostsSpec from a cloud spec disruption, get all ip ranges of services provided and transform them into a list of hosts spec
+func transformCloudSpecToHostsSpec(log *zap.SugaredLogger, cloudManager *cloudservice.CloudServicesProvidersManager, cloudSpec *v1beta1.NetworkDisruptionCloudSpec) ([]v1beta1.NetworkDisruptionHostSpec, error) {
+	hosts := []v1beta1.NetworkDisruptionHostSpec{}
+	clouds := map[cloudtypes.CloudProviderName]*[]string{
+		cloudtypes.CloudProviderAWS: cloudSpec.AWSServiceList,
+	}
+
+	for cloudName, serviceList := range clouds {
+		ipRanges, err := cloudManager.GetServiceIPRanges(cloudName, *serviceList)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, ip := range ipRanges {
+			hosts = append(hosts, v1beta1.NetworkDisruptionHostSpec{
+				Host:     ip,
+				Protocol: "tcp",
+				Flow:     "egress",
```

Comment:
I may be wrong, but I think there are some load balancers and services from AWS that accept udp packets. Also is there really no case that someone may want an ingress disruption instead?

Reply:
Thanks for catching that!